apps/content-publishing-worker/src/publisher/message.publisher.ts

```diff
@@ -9,6 +9,8 @@ import { PinoLogger } from 'nestjs-pino';
 
 @Injectable()
 export class MessagePublisher implements OnApplicationBootstrap {
+  private static readonly BATCH_PROCESS_WINDOW_MS = 6000;
+
```

Comment:
If the intent is to have the batch window line up with our expected block time, we have a way to get that value from the PolkadotApiServer: `blockTimeMs`

Reply:
The intent was to start with this but be able to adjust the value if needed.

Reply:
Ah yeah that's better, fixed

Reply:
I wonder if specifying it as a multiplier of the `blockTimeMs` might be a good way to go? I feel like, regardless of the magnitude of the value, the point of using a time interval to collect batches is that it is related to the fact that blocks are also produced on a period, so it's nice to show that relationship in the definition of the constant.

Not important; non-blocking.